File: bai 5/app/src/main/java/com/example/bth5_xulysukien1/MainActivity.java
package com.example.bth5_xulysukien1;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    EditText edtSo1, edtSo2, edtKetQua;
    Button btnCong, btnTru, btnNhan, btnChia;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        edtSo1 = findViewById(R.id.edtSo1);
        edtSo2 = findViewById(R.id.edtSo2);
        edtKetQua = findViewById(R.id.edtKetQua);

        btnCong = findViewById(R.id.btnCong);
        btnTru = findViewById(R.id.btnTru);
        btnNhan = findViewById(R.id.btnNhan);
        btnChia = findViewById(R.id.btnChia);
 //Commit
        btnCong.setOnClickListener(v -> {
            float soA = Float.parseFloat(edtSo1.getText().toString());
            float soB = Float.parseFloat(edtSo2.getText().toString());
            edtKetQua.setText(String.valueOf(soA + soB));
        });

        btnTru.setOnClickListener(v -> {
            float soA = Float.parseFloat(edtSo1.getText().toString());
            float soB = Float.parseFloat(edtSo2.getText().toString());
            edtKetQua.setText(String.valueOf(soA - soB));
        });

        btnNhan.setOnClickListener(v -> {
            float soA = Float.parseFloat(edtSo1.getText().toString());
            float soB = Float.parseFloat(edtSo2.getText().toString());
            edtKetQua.setText(String.valueOf(soA * soB));
        });

        btnChia.setOnClickListener(v -> {
            float soA = Float.parseFloat(edtSo1.getText().toString());
            float soB = Float.parseFloat(edtSo2.getText().toString());
            edtKetQua.setText(String.valueOf(soA / soB));
        });
    }
}

File: Bai2/src/main/java/com/example/bai2/MainActivity.kt
package com.example.bai2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        //Commit
        val so1 = findViewById<EditText>(R.id.edtSo1)
        val so2 = findViewById<EditText>(R.id.edtSo2)
        val btnTinh = findViewById<Button>(R.id.btnTinh)
        val ketQua = findViewById<TextView>(R.id.txtKetQua)

        btnTinh.setOnClickListener {

            val a = so1.text.toString().toIntOrNull() ?: 0
            val b = so2.text.toString().toIntOrNull() ?: 0

            val ketqua = a + b

            ketQua.text = getString(R.string.ketqua, ketqua)
        }
    }
}